modules/oxide/package.json

```diff
@@ -36,6 +37,8 @@
     "prismjs": "^1.27.0"
   },
   "devDependencies": {
-    "gulp": "^4.0.2"
+    "connect": "^3.7.0",
+    "connect-livereload": "^0.6.1",
+    "stylelint-config-recommended-less": "^3.0.1"
```

Comment:
I think these dependencies should be moved to the workspace package.json

Reply:
these are module-specific dependencies, shouldn't they stay in module `package.json`

Reply:
I believe it is fine to leave this as is as like Arvin said, these dev dependencies are specific to the module

Reply:
The open source project puts all dev dependencies at the top level.

However I stopped doing that for the premium monorepo; initially because yarn 4 required it but I think in general it makes sense to only put shared dev dependencies at the top level (because those are the ones we _really_ want to keep the versions matching across the codebase).